rotkehlchen/balances/manual.py

```diff
@@ -102,30 +103,20 @@ def remove_manually_tracked_balances(db: 'DBHandler', labels: List[str]) -> None
 
 def account_for_manually_tracked_balances(
         db: 'DBHandler',
-        balances: Dict[str, Any],
+        balances: Dict[str, Dict[Asset, Balance]],
 ) -> Dict[str, Any]:
     """Given the big balances mapping adds to it all manually tracked balances"""
     manually_tracked_balances = get_manually_tracked_balances(db)
     for m_entry in manually_tracked_balances:
         location_str = str(m_entry.location)
+        balance = Balance(
+            amount=m_entry.amount,
+            usd_value=m_entry.usd_value,
+        )
         if location_str not in balances:
-            balances[location_str] = {}
-            balances[location_str][m_entry.asset] = {
-                'amount': m_entry.amount,
-                'usd_value': m_entry.usd_value,
-            }
+            balances[location_str] = {m_entry.asset: balance}
```

Comment:
I had an implementation with `defaultdict` but what I gained in number of lines reduced I lost it in comprehension, therefore I kept it old-fashion.